.github/workflows/check-pr.yml

```
@@ -0,0 +1,17 @@
+name: Check Pull Request
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  validate-pr-label:
+    name: Validate the Pull Request's labels
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR must be labeled
+        uses: jesusvasquez333/verify-pr-label-action@v1.4.0
+        with:
+          disable-reviews: true
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          valid-labels: "breaking-change, bug, chore, ci, dependencies, documentation, enhancement, refactoring, test"
```

```suggestion
          valid-labels: "bug, chore, ci, dependencies, documentation, enhancement, refactoring"
```

i don't think we have tests or breaking changes?

---

I also thought about it at first and removed them. Then I did some more research and it seems that the labels **could** make sense after all. For example, if we add a migration guide and mix up the documentation, "breaking change" as label could be appropriate.

However, I think that the other labels such as "documentation" and "enhancement" are not so good in coexistence either in the documentation repository. I would be in favor of rethinking from scratch which labels make sense and which don't, and then revising them altogether. The same applies to the other repositories.

---

oh - right, documentation does not make sense at all in this repo as all documentation things here are enhancements!

---

I'd say let's get started with a minimum set of labels we think we need (please sync them to the settings file too) and expand it if we need more.